iio/iio_app/iio_app.c

```diff
@@ -37,24 +37,17 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 *******************************************************************************/
 
-#ifdef IIO_SUPPORT
+// #ifdef IIO_SUPPORT
```

Comment:
also this, shouldn't be commented out I guess

Reply:
Just for development time.